float32 poi_range
float32 poi_angle
int16 is_tracking